.Rbuildignore

```diff
@@ -17,3 +17,4 @@
 ^vignettes/target-market-share\.Rmd$
 ^vignettes/target-sda\.Rmd$
 ^cran-comments\.md$
+^vignettes/articles$
```

Comment:
This setup is better done via `use_article()` -- which does more than just adding an .Rmd file into vignettes.